ansible/roles/debops.resources/defaults/main.yml

```diff
@@ -92,33 +92,31 @@ resources__parent_dirs_mode: '{{ omit }}'
 # These variables define how the role will manage custom templates on remote
 # hosts. See :ref:`resources__ref_templates` for more details.
 
-# .. envvar:: resources__group_name [[[
-#
-# Name of the directory which contains templates which should be generated on
-# hosts in a specific host group. This variable needs to be set on a group
-# level in the inventory to take effect, only one group is supported at a time.
-resources__group_name: 'nonexistent-host-group'
-
-                                                                   # ]]]
 # .. envvar:: resources__templates [[[
 #
 # Directory which contains templates that should be generated on all hosts in
 # the Ansible inventory.
-resources__templates: '{{ resources__src + "templates/by-group/all" }}'
+resources__templates: [ '{{ resources__src + "templates/by-group/all" }}' ]
 
                                                                    # ]]]
 # .. envvar:: resources__group_templates [[[
 #
-# Directory which contains templates that should be generated on hosts in
-# a specific Ansible inventory group.
-resources__group_templates: '{{ resources__src + "templates/by-group/" + resources__group_name }}'
+# List of paths containing the directories of all the groups the current host is in, based on the content of `group_names`.
+# See `Ansible - Playbooks Varibles <https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#accessing-information-about-other-hosts-with-magic-variables>`_.
+#
+# For example if the host `debian1` is member of `group-name1` and `group-name2`
+# `debops` will then search all template files inside the directories placed here: :file:`ansible/resources/template/by-group/`.
```

Comment:
I would say that the `debops.resources` role would search for the template files, not the `debops` script itself.

Also, the path is misspelled - `templates`, not `template`.